docs/source/user_guide/app.rst

```diff
@@ -1056,6 +1056,16 @@ You can also programmatically configure a session's color scheme by creating
     Refer to the |ColorScheme| class for documentation of the available
     customization options.
 
+.. note::
+
+    For maskTargetsColors, intTarget must be an integer between 1 and 255. 
```

Comment:
Can we instead update the frontend to allow targets > 255? The backend should support it and that was the intent as the user could have more than 255 classes in their dataset

Reply:
Yeah, I would remove the FE constraints on > 255. Will only prune 0 and negative number.

Reply:
To clarify, are you proposing the following?
- Add validation when saving color schemes so that invalid things like negative/0 will raise an error?
- If, for some reason, bad data got into a color scheme, ignore it in the frontend? IE ignore negative/0 values?

Reply:
The confusion is mostly from SDK users, so I am proposing to: 
- add validation methods when we initiate the `ColorScheme` Class, so when a user enter bad input in the python SDK, they can see error messages immediately.
- in the FE, do not allow new bad inputs. For existing bad inputs, still display the value as is, but display a small warning text.